runtime/local/src/precompiles.rs

```diff
@@ -84,6 +84,7 @@ impl Contains<RuntimeCall> for WhitelistedLockdropCalls {
                 ..
             }) => true,
             RuntimeCall::Balances(pallet_balances::Call::transfer { .. }) => true,
+            RuntimeCall::Balances(pallet_balances::Call::transfer_keep_alive { .. }) => true,
```

Comment:
I'm not sure this is the correct approach - I don't have any issue with the call, but we want to get rid of this account usage. So I'd expect portal team to ask for `transfer_all`, not the one that keeps the account alive 😁

Reply:
That's what I've done, and also think it's the best way.

Reply:
I'd still suggest to also add `transfer_allow_death` to keep the full set of _transfer_ functions available.

Up to you though, I'll approve.

Reply:
I share the same thoughts as Dino, might as well allow all of transfer functions.